include/TPP/Dialect/VNNI/VNNIOps.td

```diff
@@ -0,0 +1,45 @@
+//===- VNNIOps.td - VNNI dialect ops -----------------------*- tablegen -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+
+#ifndef VNNI_DIALECT_VNNI_OPS
+#define VNNI_DIALECT_VNNI_OPS
+
+include "mlir/IR/OpBase.td"
+
+def VNNI_Dialect : Dialect {
+  let name = "vnni";
+  let cppNamespace = "::mlir::vnni";
+  let summary = [{
+    A dialect implementing Intel Vector Neural Network Instructions operations.
+    https://www.intel.com/content/www/us/en/artificial-intelligence/deep-learning-boost.html
+  }];
+}
+
+def VNNI_MatmulOp : Op<VNNI_Dialect, "matmul", [TypesMatchWith<"result type matches type of dest",
```

Comment:
can you format a bit better here? maybe `[TypesMatchWith<"result type matches type of dest"` this should go in a new line.